.bazelrc

```diff
@@ -72,6 +72,9 @@ build:clang-asan --linkopt -fuse-ld=lld
 
 # macOS ASAN/UBSAN
 build:macos --cxxopt=-std=c++17
+build:macos --action_env=BAZEL_LINKLIBS=-l%:libstdc++.a
+build:macos --action_env=BAZEL_LINKOPTS=-lm
```

Comment:
I don't think macOS need this either.

Reply:
Resolving in https://github.com/envoyproxy/envoy/pull/15927 - thanks for the action item.